contrib/babelfishpg_tds/src/backend/tds/tdsresponse.c

```diff
@@ -1725,11 +1725,15 @@ PrepareRowDescription(TupleDesc typeinfo, List *targetlist, int16 *formats,
 												att->attcollation, (atttypmod - 4) * 2);
 				break;
 			case TDS_SEND_VARCHAR:
+				if (atttypmod == TSQLMaxTypmod)
```

Comment:
Why cant we update this in the source of the typmod rather than in the protocol which a superficial conversion